mopro-ffi/Cargo.toml

```diff
@@ -14,7 +14,8 @@ name = "uniffi-bindgen"
 path = "uniffi-bindgen.rs"
 
 [features]
-default = []
+# default = []
+default = ["gpu-benchmarks"]
```

Comment:
Not sure we want to keep this on by default, might interfere with other Anon-Aadhaar integration work

Reply:
Sure, we can keep the default clean and comment on the one with "gpu-benchmarks" feature at this moment.

We found that the `mopro-ffi/tests/bindings` tests only take the default features in `mopro-ffi/Cargo.toml`. If we can find a way to have the tests in bindings took `--feature` flags, we will remove the `# default = ["gpu-benchmarks"]`.